src/dcc/LocalTrackIf.cxx

```diff
@@ -36,10 +36,16 @@
 #include <unistd.h>
 #include <fcntl.h>
 
+#include "openmrn_features.h"
+#ifdef OPENMRN_FEATURE_FD_CAN_DEVICE
```

Comment:
> I don't know what fd CAN has to do with the DCC local track device.

This define is used as part of omitting the calls to `ioctl` with the extra flags that are defined in can_ioctl.h, it can probably be reviewed further.

> since the esp32 does not have a DCC driver via fd

I've implemented one [here](https://github.com/atanisoft/ESP32CommandStation/blob/master/components/DCC/DCCSignalVFS.cpp) but it has dependencies on the RMT peripheral and can create problems for other usages of the peripheral due to hooking into the RMT TX complete callback. Due to the RMT conflicts I have not proposed including it in openmrn.